README.md

```diff
@@ -84,8 +84,8 @@ curl base_url/user/create/ \
 
 ### Create a Community
 ```
-curl base_url/community/create/ \
-    -X POST \
+curl base_url/community/ \
+    -X PUT \
```

Comment:
If this is meant to be the endpoint for creating a user, it should be a `POST` request

Reply:
Sure ✌️. Ill make that change.

Reply:
The `create_user` and `create_community` methods are made to be accessed only by us (they wont be accessible to the public), so do we really need to follow conventions for these 2 methods ?

Reply:
It's always important to stick to conventions and be consistent through the application, so we should stick to the same conventions